engine/Assets/Scripts/UI/Dynamic/MainHUD.cs

```diff
@@ -164,4 +174,118 @@ public static class MainHUD {
+        MainHUD.AddItemToDrawer("Scoring Zones", b => {
+            if (FieldSimObject.CurrentField == null) {
+                SynthesisAPI.Utilities.Logger.Log("No field loaded!", LogLevel.Info);
+            } else {
+                if (!DynamicUIManager.PanelExists<ScoringZonesPanel>())
+                    DynamicUIManager.CreatePanel<ScoringZonesPanel>();
+            }
+        });
+        MainHUD.AddItemToDrawer("Camera View", b => DynamicUIManager.CreateModal<ChangeViewModal>(),
+            icon: SynthesisAssetCollection.GetSpriteByName("CameraIcon"));
+        MainHUD.AddItemToDrawer("Download Asset", b => DynamicUIManager.CreateModal<DownloadAssetModal>(),
+            icon: SynthesisAssetCollection.GetSpriteByName("DownloadIcon"));
+        MainHUD.AddItemToDrawer("DriverStation",
+            b => DynamicUIManager.CreatePanel<BetaWarningPanel>(
+                false, (Action) (() => DynamicUIManager.CreatePanel<DriverStationPanel>(true))),
+            icon: SynthesisAssetCollection.GetSpriteByName("driverstation-icon"));
+        MainHUD.AddItemToDrawer("Settings", b => DynamicUIManager.CreateModal<SettingsModal>(),
+            icon: SynthesisAssetCollection.GetSpriteByName("settings"));
+
+        PhysicsManager.IsFrozen = false;
+    }
+
+    public static void SetUpMatch() {
+        foreach (string name in MainHUD.DrawerTitles)
+            MainHUD.RemoveItemFromDrawer(name);
+
+        if (RobotSimObject.CurrentlyPossessedRobot != string.Empty)
+            MainHUD.AddItemToDrawer(
+                "Configure", b => SetUpConfig(), icon: SynthesisAssetCollection.GetSpriteByName("wrench-icon"));
+
+        MainHUD.AddItemToDrawer("Multibot", b => DynamicUIManager.CreatePanel<RobotSwitchPanel>());
+        MainHUD.AddItemToDrawer("Camera View", b => DynamicUIManager.CreateModal<ChangeViewModal>(),
+            icon: SynthesisAssetCollection.GetSpriteByName("CameraIcon"));
+        MainHUD.AddItemToDrawer("Settings", b => DynamicUIManager.CreateModal<SettingsModal>(),
+            icon: SynthesisAssetCollection.GetSpriteByName("settings"));
+        MainHUD.AddItemToDrawer("DriverStation",
+            b => DynamicUIManager.CreatePanel<BetaWarningPanel>(
+                false, (Action) (() => DynamicUIManager.CreatePanel<DriverStationPanel>(true))),
+            icon: SynthesisAssetCollection.GetSpriteByName("driverstation-icon"));
+        MainHUD.AddItemToDrawer("Scoring Zones", b => {
+            if (FieldSimObject.CurrentField == null) {
+                SynthesisAPI.Utilities.Logger.Log("No field loaded!", LogLevel.Info);
+            } else {
+                if (!DynamicUIManager.PanelExists<ScoringZonesPanel>())
+                    DynamicUIManager.CreatePanel<ScoringZonesPanel>();
+            }
+        });
+
+        if (!(MatchStateMachine.Instance.CurrentState.StateName is MatchStateMachine.StateName.RobotPositioning))
+            PhysicsManager.IsFrozen = false;
```

Comment:
I'm running into an issue where robots are falling through the floor when I go from placement to score zone setup in match mode. This is likely the issue. PhysicsManager.IsFrozen does not work like a normal boolean and is essentially a counter. Assigning it to false like that will likely cause whatever needs to pause it to fail to actually pause.